srv/salt/ceph/salt/crc/default.sls

```diff
@@ -1,4 +1,4 @@
-/etc/salt/minion:
+/etc/salt/master.d/crc_method:
```

Comment:
Nit: all our config fragments in [master,minion].d have a `.conf` extension. Maybe we should keep things consistent?

Reply:
ack